synapse/tests/test_lib_layer.py

```diff
@@ -661,35 +661,17 @@ async def test_layer_stortype_merge(self):
             nodes = await core.nodes('inet:ipv4=1.2.3.4 [ +#foo.bar=2015 ]')
             self.eq((1325376000000, 1420070400001), nodes[0].getTag('foo.bar'))
 
-            nodeedits = [
```

Comment:
IIRC this tested applying node edits and showing that the results of applying edits without changes resulted in a empty set of changes. I guess this would be OBE with the sode format change.